utils/supabase/getSession.ts

```diff
@@ -0,0 +1,27 @@
+"use server";
+import { cookies } from "next/headers";
+
+export const getSession = async () => {
+  const cookieStore = cookies();
+  const session = cookieStore.get("session");
+
+  if (session) {
+    try {
+      const sessionData = JSON.parse(session.value);
+
+      const { access_token, expires_at, refresh_token } = sessionData;
+
+      const expiresAtDate = new Date(expires_at * 1000);
```

Comment:
What does 1000 represent?

Reply:
We should then create a constant on top of this file and add comment to explain that here.